src/commands/admin/resetlevels.ts

```diff
@@ -56,7 +52,9 @@ export default class extends Command {
 				}
 			});
 
-		return loadingMsg.edit('I\'ve reset all levels in the guild.');
+		await interaction.editReply('I\'ve reset all levels in the guild.');
```

Comment:
Why not `return interaction.editReply(...)`?

Reply:
copy ben code oopsie